### PR TITLE
New vignette: Themes

### PR DESCRIPTION
**Pull request by vincentarelbundock**
A short themes vignette showing ultra-minimal examples of customization for MkDocs and Docsify, as well as links to user guides for each documentation engine.

### CODE REVIEW

File: docs/vignettes/themes.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+stores the basic settings that `altdoc` uses to render the documentation
+website. To customize your documentation website, you can edit the
+settings files stored in the `altdoc/` directory before calling
+`render_docs()`.
+
+Note that the contents of `altdoc/` are never modified when you update
+the website with `render_docs()`. Therefore, you can safely edit the
+settings files by hand to customize your website. (Warning: `altdoc/`
+can be overwritten if you call `setup_docs()` again with the
+`overwrite=TRUE` argument.)
```
Comment (etiennebacher) [inline docs/vignettes/themes.md:13]:
```suggestion
can be overwritten if you call `setup_docs()` again with 
`overwrite=TRUE`.)
```

File: docs/vignettes/themes.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+``` r
+render_docs()
+```
+
+Here is a preview of the result:
+
+![](images/themes_docsify_preview.png)
+
+``` {html}
```
Comment (etiennebacher) [inline docs/vignettes/themes.md:66]: can you put this is a `<details>` tag?

**Review by etiennebacher — APPROVED** [review]
Thanks, looks good! Do you plan on adding an example for docute as well?

File: docs/vignettes/themes.md (hunk 2 above)
Comment (vincentarelbundock) [reply docs/vignettes/themes.md:70]: https://github.com/etiennebacher/altdoc/pull/95/commits/2cdb37f7ad4488577d045a522293853590f13341

**Comment by vincentarelbundock** [comment]
> Thanks, looks good! Do you plan on adding an example for docute as well?

Not for now.

My understanding is that Docute is no longer maintained, so I don't think we should do too much to encourage users in that direction, and I'm less personally interested in spending time on docute-related features.

image

File: docs/vignettes/themes.qmd [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+Here is a preview of the result:
+
+![](images/themes_docsify_preview.png)
+
+
+Click on Details to display the full HTML template:
+
+<details>
```
Comment (etiennebacher) [inline docs/vignettes/themes.qmd:52]: I think it also requires a `<summary>` tag for the text of the button, like "click here"

https://www.w3schools.com/TAGS/tag_details.asp

Comment (etiennebacher) [reply docs/vignettes/themes.qmd:52]: really not sure about this actually

Comment (vincentarelbundock) [reply docs/vignettes/themes.qmd:52]: I rendered it and what this shows currently is the text above, and then on the next line you have an arrow with the word "Details".

I think it's pretty clear currently, but I'm not an HTML guru.

Comment (etiennebacher) [reply docs/vignettes/themes.qmd:52]: Ah ok, fine then, I just didn't see the rendered output

**Pull request merged**